bin.js

```diff
@@ -77,7 +77,7 @@ vendorPromise.then(function () {
 					console.error(err || stats.toString());
 					return;
 				}
-				console.log('built');
+				console.log('build');
```

Comment:
build ~ process of building
built ~ process of building has finished

so this sould be `built`